### PR TITLE
Making @IBOutlets private

### PR DESCRIPTION
**Pull request by chrisfsampaio**
- Adding the `private` keyword for `@IBOutlets`
- Adding menu items to toggle the features offered by the plugin
- Adding an example project to test things out

🚀 Make Xcode Great Again™

### CODE REVIEW

File: OptionalOutlets/NSString+OptionalOutlets_Additions.m [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#import "NSString+OptionalOutlets_Additions.h"
+
+@implementation NSString (OptionalOutlets_Additions)
+
+- (nonnull NSString *)replaceImplicitlyUnwrappedForOptional {
+    return [self replaceMatchesWithRegex:@"^(\\s*@IBOutlet[\\w:\\s]*)!" replaceTemplate:@"$1?"];
+}
+
+- (nonnull NSString *)addPrivateKeyword {
+    return [self replaceMatchesWithRegex:@"(^\\s*@IBOutlet)\\sweak\\s([\\w:\\s]*)" replaceTemplate:@"$1 private weak $2"];
```
Comment (fpg1503) [inline OptionalOutlets/NSString+OptionalOutlets_Additions.m]: It only works with `weak` outlets, shouldn't we make it work with `strong` outlets as well?

Comment (fpg1503) [inline OptionalOutlets/NSString+OptionalOutlets_Additions.m]: I may be overcomplicating it but I think replacing it with `return [self replaceMatchesWithRegex:@"(^\\s*@IBOutlet)\\s*((?:weak\\s*)?\\s*var\\s*\\w*:[\\w\\s]*)" replaceTemplate:@"$1 private $2"];` fixes it.

If I don't add `var` to the regex it loops adding `private` many times when it's `strong` 🙈

**Comment by chrisfsampaio** [comment]
👍 33b10cad2bd6297332b82568dfe9a6ee16b6d258

**Comment by fpg1503** [comment]
LGTM :+1:

**Pull request merged**